monocle/db/queries.py

```diff
@@ -77,6 +71,7 @@ def generate_filter(repository_fullname, params):
     on_authors = params.get('on_authors')
     exclude_authors = params.get('exclude_authors')
     created_at_range = {"created_at": {"format": "epoch_millis"}}
+    change_ids = params.get('change_ids')
```

Comment:
Because the params is now handled there, that is ok.

Reply:
ok